draft-ietf-trans-rfc6962-bis.md

```diff
@@ -178,13 +175,17 @@ operation is asynchronous to allow clients to proceed without delay, despite
 possible issues such as network connectivity and the vagaries of firewalls.
 
 The append-only property of each log is achieved using Merkle Trees, which can
-be used to show that any particular instance of the log is a superset of any
-particular previous instance. Likewise, Merkle Trees avoid the need to blindly
-trust logs: if a log attempts to show different things to different people, this
-can be efficiently detected by comparing tree roots and consistency proofs.
-Similarly, other misbehaviors of any log (e.g., issuing signed timestamps for
-certificates they then don't log) can be efficiently detected and proved to the
-world at large.
+be used to efficiently prove that any particular instance of the log is a
+superset of any particular previous instance and to efficiently detect various
+misbehaviors of the log (e.g., issuing a signed timestamp for a certificate that
+is not subsequently logged).
+
+It is necessary to treat each log as a trusted third party, because the log
+auditing mechanisms described in this document can be circumvented by a
+misbehaving log that shows different, inconsistent views of itself to different
+clients. Whilst it is anticipated that additional mechanisms will be developed
```

Comment:
s/will/could
Developing such mechanisms is difficult and I certainly haven't seen one described in full anywhere.
So let's avoid the presentation as if it is inevitable.